### ISSUE TITLE
[security/attestation] The ownership state shall be encoded in the attestation measurements.

### ISSUE THREAD

**Issue by moidx** [lowRISC/opentitan#3255]
As part of the ownership unlock operation, `ROM_EXT` verifies the unlock command and updates the device state to `UNOWNED`. The device proceeds with the boot flow and reports the unlock result to the kernel via `BOOT_SERVICES` shared memory. 

This issue is tracks any attestation updates required as a result of this change. The current proposal is to capture the ownership state in the attestation measurements, so that it is easier to verify the state of the device. 

The kernel operating in `UNOWNED` mode may request an attestation update from the `ROM_EXT` via `BOOT_SERVICES`.

**Comment by tjaychen** [comment]
@moidx , just to confirm, this should mean that the non-volatile memory containing owner state should NOT be modifiable by software downstream of ROM_EXT right?

At the moment, are you thinking about storing this in general data partition or info partition?

**Comment by moidx** [comment]
@tjaychen, that is correct. We are currently planning to store this data in an info partition. 

+@gkelly
